services/apps/alcs/src/providers/typeorm/migrations/1670636483523-update_document_table.ts

```diff
@@ -0,0 +1,32 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class updateDocumentTable1670636483523 implements MigrationInterface {
+  name = 'updateDocumentTable1670636483523';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."document" ADD "source" character varying NOT NULL`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."document" DROP CONSTRAINT "FK_9b9254f412e8a5a07063209a08c"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "alcs"."document" ALTER COLUMN "uploaded_by_uuid" DROP NOT NULL`,
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
```

Comment:
Why is the down function dropping the same constraint that up also drops?

Reply:
That is a good typeorm question 🙂 I'll check if I can regenerate it and if not I'll drop it